build.gradle

```diff
@@ -9,6 +9,7 @@ plugins {
   id 'checkstyle'
   id 'pmd'
   id 'jacoco'
+  id 'idea'
```

Comment:
Author note:
Just adding the plugin won't create any metadata for either IDE until you run the specific tasks. So no harm. This is just ignored by other IDEs